components/bandwidth/bandwidth.app.js

```diff
@@ -1,4 +1,59 @@
+const bandwidthMessaging = require("@bandwidth/messaging");
+
 module.exports = {
   type: "app",
   app: "bandwidth",
+  propDefinitions: {
+    messageTo: {
+      type: "string",
+      label: "To",
+      description: "The number the message will be sent to, in E164 format ex `+19195551234`.",
+    },
+    from: {
+      type: "string",
+      label: "From",
+      description: "The number the call or message event will come from, in E164 format ex `+19195551234`.",
```

Comment:
Will these numbers always be tied to a Bandwidth customer's account, or can the user enter any number here?

Reply:
For messaging - yes

For outbound phone calls - no, users can create outbound calls from any number to give the appearance of an automated call coming from someones main office line, so if the end user calls back they are directed to the office line as opposed to calling a tn on bandwidth's network

Reply:
Great thanks. Does it make sense to create one prop that lists numbers in the user's account that you can use for messaging sources / actions, and use this field as a freeform text prop that you can use for phone components?

In general, we always prefer listing information tied to the user's account via API, if possible. This lets the user choose items from a list without having to look them up, and there's no room for mistaken entry. We've found it dramatically improves the usability of components, so that's why I'm asking so many questions about the API functionality.